src/main/java/thewaypointers/trafficsimulator/TrafficSimulatorManager.java

```diff
@@ -1,76 +0,0 @@
-package thewaypointers.trafficsimulator;
-
-import thewaypointers.trafficsimulator.common.IStateProvider;
-import thewaypointers.trafficsimulator.common.WorldStateDTO;
-import thewaypointers.trafficsimulator.common.helpers.JunctionTestProvider;
-import thewaypointers.trafficsimulator.gui.MainFrame;
-
-import javax.swing.*;
-
-public class TrafficSimulatorManager extends Thread {
-
-    public static final float VEHICLE_MOVEMENT_SPEED = 2;
-    public static final float STATES_PER_SECOND = 1;
-    private static MainFrame mainFrame;
-    private static IStateProvider simulation;
```

Comment:
Why were those static? There was no need for them to be static, and it caused problems.